native/src/components/PoiFiltersModal.tsx

```diff
@@ -55,11 +56,11 @@ const FlexEnd = styled.View`
   justify-content: flex-end;
 `
 
-const TileRow = styled(Row)`
-  place-content: space-between center;
-  flex-wrap: wrap;
-  gap: 16px;
-`
+// const TileRow = styled(Row)`
+//   place-content: space-between center;
+//   flex-wrap: wrap;
+//   gap: 16px;
+// `
```

Comment:
You can remove the commented out code.

Reply:
Does Nexus 6 emulation let you scroll?
hmm.. so do you prefer to see the whole buttons on small devices and make the scrolling on the entire page?